app/furniture/marketplace/delivery_area_policy.rb

```diff
@@ -9,7 +9,7 @@ def create?
     alias_method :update?, :create?
 
     def permitted_attributes(_)
-      [:label, :price, :order_by, :delivery_window]
+      [:label, :price, :order_by, :delivery_window, :restore]
```

Comment:
Is it ruby/rails syntax convention to horizontally stack lists? I guess I've gotten used to left-aligned vertical lists for collections so I find them easier to scan that way. 😅

Reply:
Eh, it's kind of a "six in one half dozen in the other" for me. I can imagine there's legibility benefits to vertical lists rather than horizontal ones; and am happy to receive feedback on a case-by-case basis for where it's useful.

Reply:
I'll think about it 🤨